### PR TITLE
Update charter.md

### PR DESCRIPTION
**Pull request by jimsch**
* Deal with all of Barry's nits in #3
* Deal with first and third of Martin's Comments #8

### CODE REVIEW

**Comment by fpalombini** [comment]
@jimsch I am done, take a look and see if it's ok. The only comment that was not addressed was Ben's second comment: 

> I found the wording confusing in some parts of the charter, such as "collect these features", the treatment of CDDL evolution as a "sequence of editions"

File: charter.md [hunk 1]
```diff
@@ -37,30 +35,17 @@ considered precious, and the WG will set out not to damage their value.
...
-decide to adopt these proposals, but typically it would just provide input
-and recommend that they be published either as an Independent Submission or
-by a different working group.
-
-*   Narrow purpose specifications: The working group may provide evaluation
-of such proposals, but typically would not support Working Group adoption,
-and could recommend publication in a different forum. An example of this
-might be portions of draft-bormann-cbor-tags-oid dealing with some of the
-more esoteric types such as regular expressions.
+The working group will evaluate such proposal individually and decide about
```
Comment (cabo) [inline charter.md]: proposal_s_

File: charter.md [hunk 2]
```diff
@@ -37,30 +35,17 @@ considered precious, and the WG will set out not to damage their value.
...
-and recommend that they be published either as an Independent Submission or
-by a different working group.
-
-*   Narrow purpose specifications: The working group may provide evaluation
-of such proposals, but typically would not support Working Group adoption,
-and could recommend publication in a different forum. An example of this
-might be portions of draft-bormann-cbor-tags-oid dealing with some of the
-more esoteric types such as regular expressions.
+The working group will evaluate such proposal individually and decide about 
+addition of a respective milestone. Proposals that are deemed to be out of
```
Comment (cabo) [inline charter.md]: the addition?

**Review by cabo — APPROVED** [review]
Couple of grammos, but looks good otherwise

**Pull request merged**